packages/playground/_stories/main/RangeSlider/RangeSlider.stories.ts

```diff
@@ -35,38 +35,54 @@ const Template: UI5StoryArgs<RangeSlider, StoryArgsSlots> = (args) => html`
 	?show-tooltip="${ifDefined(args.showTooltip)}"
 ></ui5-range-slider>`;
 
-export const BasicRangeSlider = Template.bind({});
-BasicRangeSlider.args = {
+export const Basic = Template.bind({});
+Basic.decorators = [
+	(story) => {
+		return html`
+			<div class="wrapper" style="margin-top:1rem;">
```

Comment:
There are identation issues 
![image](https://github.com/SAP/ui5-webcomponents/assets/7827314/b7efbdd3-1d2e-4b00-a4d9-eb1662228e20)

Actually could this be placed in the const Template only once?